docs/commands.rst

```diff
@@ -0,0 +1,85 @@
+.. _commands:
+
+Commands
+========
+
+``django-pgtrigger`` comes with the ``manage.py pgtrigger`` command. It
+can be called with the following subcommands.
+
+ls
+--
+
+Use ``manage.py pgtrigger ls`` to list all triggers that are actively
+managed by ``django-pgtrigger``. The trigger URI, installation status,
+and whether the trigger is enabled or disabled will be shown.
+
+The following are valid installation status markers:
+
+- ``INSTALLED``: The trigger is installed and up to date
+- ``OUTDATED``: The trigger is installed, but it is outdated. New changes
+  need to be applied with ``manage.py pgtrigger install``.
+- ``UNINSTALLED``: The trigger is not active and is not yet installed.
+- ``PRUNE``: A trigger that was formerly installed by ``django-pgtrigger``
+  is no longer in the codebase and needs to be pruned. Pruning can be done
+  with ``manage.py pgtrigger prune`` or by doing a full
+  ``manage.py pgtrigger install`` or ``manage.py pgtrigger uninstall``.
+  Note that these triggers are still active and running and are no different
+  than installed triggers.
+
+Note that every installed trigger, including ones that will be pruned,
+will show whether they are enabled or disabled. Disabled triggers are
+installed but do not run, and they have to be manually re-enabled.
+Enabling and disabling of triggers can be performed with
+``manage.py pgtrigger enable`` and ``manage.py pgtrigger disable``.
+
+.. note::
+
+  You can provide trigger URIs as arguments to ``manage.py pgtrigger ls``
+  to only list specific triggers. The URI is the first column returned
+  by ``manage.py pgtrigger ls``.
+
+install
+-------
+
+Use ``python manage.py install`` to install triggers. If no arguments are
+provided, it will try to install all triggers that are not currently installed.
+Messages will be printed for every installed trigger. When executed with
+no arguments, any triggers that were previously installed by ``django-pgtrigger``
+but no longer in the codebase will be pruned.
+
+You can provide trigger URIs as arguments to install specific triggers.
+Trigger URIs can be found with the ``manage.py pgtrigger ls`` command.
+
+uninstall
+---------
+
+Uninstall has the same behavior as ``manage.py pgtrigger install`` except triggers
+will be uninstalled.
+
+enable
+------
+
+``manage.py pgtrigger enable`` will enable all triggers. All triggers
+that were previously disabled will be printed when enabled. In contrast
+to ``manage.py pgtrigger install``, no previously managed triggers will
+be pruned.
+
+Similar to other commands, one can provide trigger URIs as arguments.
+Trigger URIs can be found with the ``manage.py pgtrigger ls`` command.
+
+disable
+-------
+
+Disable has the same behavior as ``manage.py pgtrigger enable`` except triggers
+will be disabled.
+
+prune
```

Comment:
Not sure if this would be a place to put it - but I take it that this version of prune will remove all "legacy" pg-triggers installed in earlier versions?

Or is the expectation that if ppl mean to drop some triggers instead of adding a name to them, they should manually go in and delete/drop those older/legacy triggers?

Reply:
To expand on my comment - after reading thru the code below - it's my understanding that you're still very much using `pgtrigger_` as a prefix (or "namespace") for all the triggers, so presumalby, all the legacy triggers will still be picked up by `prune` and dropped right?

Reply:
@tomage correct, if people update their triggers to have names which in turn changes the name of the underlying trigger, their old triggers will be "pruned" (i.e. uninstalled) since they are no longer recognized. The re-named triggers will be installed. Since the namespace name "pgtrigger_" wasn't changed, there should be no circumstance in which a random trigger is still firing.